src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -1429,6 +1503,87 @@ await using (var context = InMemoryApplicationDbContext(contextId))
             }
         }
 
+        [Fact]
+        public async Task GetInternalReleaseNotes()
+        {
+            var release = new Release
+            {
+                ReleaseStatuses = new List<ReleaseStatus>
+                {
+                    new ReleaseStatus
+                    {
+                        InternalReleaseNote = "Note1 - old",
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                        Created = new DateTime(2001, 1, 1),
+                        CreatedBy = new User { Email = "note1@test.com" }
+                    },
+                    new ReleaseStatus
+                    {
+                        InternalReleaseNote = "Note2 - latest",
+                        ApprovalStatus = ReleaseApprovalStatus.HigherLevelReview,
+                        Created = new DateTime(2002, 2, 2),
+                        CreatedBy = new User { Email = "note2@test.com" }
+                    },
+                    new ReleaseStatus
+                    {
+                        InternalReleaseNote = "Note3 - null created",
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                        Created = null, // A migrated ReleaseStatus has null Created and CreatedBy
+                        CreatedBy = null
+                    }
+                }
+            };
+            var ignoredRelease = new Release
+            {
+                ReleaseStatuses = new List<ReleaseStatus>
+                {
+                    new ReleaseStatus
+                    {
+                        InternalReleaseNote = "Note4 - different release",
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                        Created = new DateTime(2012, 12, 12),
+                        CreatedBy = new User { Email = "note4@test.com" }
+                    }
+                }
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            {
+                await contentDbContext.AddRangeAsync(release, ignoredRelease);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            {
+                var releaseService = BuildReleaseService(contentDbContext);
+                var result = await releaseService.GetReleaseStatuses(release.Id);
+                Assert.True(result.IsRight);
```

Comment:
There's a new `result.AssertRight()` extension coming from the SOW4 branch soon 🙂